internal/scheduler/jobdb/jobdb.go

```diff
@@ -59,30 +56,22 @@ func (_ RealUUIDProvider) New() uuid.UUID {
 func NewJobDb(priorityClasses map[string]types.PriorityClass, defaultPriorityClassName string, stringInterner *stringinterner.StringInterner) *JobDb {
 	return NewJobDbWithSchedulingKeyGenerator(
 		priorityClasses,
-		defaultPriorityClassName,
 		schedulerobjects.NewSchedulingKeyGenerator(),
 		stringInterner,
 	)
 }
 
 func NewJobDbWithSchedulingKeyGenerator(
 	priorityClasses map[string]types.PriorityClass,
-	defaultPriorityClassName string,
 	skg *schedulerobjects.SchedulingKeyGenerator,
 	stringInterner *stringinterner.StringInterner,
 ) *JobDb {
-	defaultPriorityClass, ok := priorityClasses[defaultPriorityClassName]
```

Comment:
Does something prevent invalid config to protect us here?

Reply:
it's a fair point.  Let me see if I can put something like this back.